project.ncl

```diff
@@ -7,7 +7,7 @@ let import_nix = organist.lib.import_nix in
     organist.shells.Bash
     & {
       build.packages = {
-        nickel-lang-cli = import_nix "nickel#nickel-lang-cli",
+        nickel-lang-cli = import_nix "nixpkgs#nickel",
         parallel = import_nix "nixpkgs#parallel",
         gnused = import_nix "nixpkgs#gnused",
       },
```

Comment:
Also, https://github.com/nickel-lang/organist/pull/142 added another reference to `nickel#lsp-nls` here.